<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryWindow</class>
 <widget class="QDialog" name="HistoryWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1133</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game History</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="replay_2_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>390</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_2_0">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>390</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_1_1">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>230</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_0_0">
   <property name="geometry">
    <rect>
     <x>619</x>
     <y>80</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="historyList">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Times New Roman&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; </string>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_2_1">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>390</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_0_1">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>80</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_0_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>80</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replayButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>710</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
   </property>
   <property name="text">
    <string>Replay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearHistoryButton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>710</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
   </property>
   <property name="text">
    <string>Clear History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_1_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>230</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="replay_1_0">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>72</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 72pt &quot;The Sunday&quot;;
    background-color: transparent;
    border: none;
    color: #FFFFFF; /* Text color */
}

QPushButton:hover {
    background-color: transparent;
    border: none;
    color: #CCCCCC; /* Optional: change text color slightly on hover */
}

QPushButton:pressed {
    background-color: transparent;
    border: none;
    color: #AAAAAA; /* Optional: darker text when pressed */
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>1131</width>
     <height>871</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/HistoryPhoto.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="backToModeButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>710</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>The Sunday</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton 
{ 
background-color:  #030554;
      color: white;
        font-size: 20px;
	font: 14pt &quot;The Sunday&quot;;
        border-radius: 30px;
        padding: 10px 20px
}</string>
   </property>
   <property name="text">
    <string>Game Mode</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>replay_2_2</zorder>
  <zorder>replay_2_0</zorder>
  <zorder>replay_1_1</zorder>
  <zorder>replay_0_0</zorder>
  <zorder>historyList</zorder>
  <zorder>replay_2_1</zorder>
  <zorder>replay_0_1</zorder>
  <zorder>replay_0_2</zorder>
  <zorder>replayButton</zorder>
  <zorder>clearHistoryButton_2</zorder>
  <zorder>replay_1_2</zorder>
  <zorder>replay_1_0</zorder>
  <zorder>backToModeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
